packages/cli/src/commands/push.ts

```diff
@@ -76,7 +78,10 @@ async function action(options: any): Promise<void> {
     ...promptDeployDependencies,
   };
 
-  if (!options.skipTelemetry) await Telemetry.report('push', pushArguments, interactive);
+  if (contractAlias) pushArguments.contractAliases = [contractAlias];
+
+  if (!options.skipTelemetry)
+    await Telemetry.report('push', (pushArguments as unknown) as Record<string, unknown>, interactive);
```

Comment:
I'm curious: why was this cast needed when changing from `{ [key: string]: unknown }` to `Record<string, unknown>` in the `report` method definition? I thought that both were semantically the same.

Reply:
The cast is needed not because of the change of `report` method but because of change `pushParams` to type `PushParams` which doesn't have required index signature.